ee/vellum_ee/workflows/display/tests/workflow_serialization/test_basic_code_execution_node_serialization.py

```diff
@@ -750,3 +752,30 @@ def test_serialize_workflow__try_wrapped():
             "try_workflow",
         ],
     }
+
+
+def test_serialize_workflow__code_execution_node_with_non_existent_filepath():
```

Comment:
this test looks great. Now lets update our implementation logic to get it to pass